matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/common/TestConstants.kt

```diff
@@ -23,7 +23,7 @@ object TestConstants {
     const val TESTS_HOME_SERVER_URL = "http://10.0.2.2:8080"
 
     // Time out to use when waiting for server response.
-    private const val AWAIT_TIME_OUT_MILLIS = 30_000
```

Comment:
I found myself doing this in response to slow integration tests in #5459 - do you have any more information on why 60s is better?

Reply:
When doing test with several sessions everything is getting a lot slower (like E2E sanity test), each session is slower to sync, etc...
Everytime we test something with await/waitWithLatch it's using that as default timeout, and test are then failing when waiting from something coming back from the sync.
It also happens when running all tests, I wonder if we don't clean properly between test?
Previously I used the `ANDROIDX_TEST_ORCHESTRATOR` option to ensure that all test start with a clear state, but looks like it's not working anymore.

Reply:
We definitely don't clean up the server between tests (and if you just run a demo server locally, then that won't be cleaned up unless you explicitly do so)